Documentation/user-guides/webhook.md

```diff
@@ -79,7 +118,8 @@ webhooks:
 
 ### `/admission-prometheusrules/mutate`
 
-The endpoint `/admission-prometheusrules/mutate` ensures that integers and boolean yaml data elements are cooerced into strings.
+The endpoint `/admission-prometheusrules/mutate` ensures that integers and
+boolean yaml data elements are coerced into strings.
```

Comment:
Although it is already clear to me, perhaps since for the `/validate` endpoint, you mentioned explicitly it is for `PrometheusRule`, for consistency we might do the same?

Reply:
good point!